server/py/services/api/api/endpoints/nuclio.py

```diff
@@ -271,6 +271,7 @@ async def deploy_function(
         function,
         model_endpoint_creation_task_name,
         returned_background_tasks,
+        _,
```

Comment:
If you agree with the prev comment you can delete this also

Reply:
We don't need the UID list in nuclio, but we need it for the job.